bundles/org.openhab.core.audio/src/main/java/org/openhab/core/audio/AudioManager.java

```diff
@@ -19,6 +19,8 @@
 import org.eclipse.jdt.annotation.Nullable;
 import org.openhab.core.library.types.PercentType;
 
+import io.reactivex.annotations.NonNull;
```

Comment:
Hi, I just briefly looked into this and I am wondering why you import an additional annotation framework.
As the main class is annotated with @ NonNullByDefault, you may not need this anyway.
Did I overlook something?

Reply:
@holgerfriedrich You are right, but it is a bad habit of making things explicit in method signatures that comes into play.

Reply:
@kgoderis The import uses a different framework, I would expect `org.eclipse.jdt.annotation.NonNull` instead of `io.reactivex.annotations.NonNull`....